src/serlo-editor/plugins/text/plugins/with-blanks.ts

```diff
@@ -0,0 +1,15 @@
+import { Editor } from 'slate'
+
+export const withBlanks = (editor: Editor) => {
+  const { isInline } = editor
+
+  editor.isInline = (element) => {
+    return element.type === 'blank' ? true : isInline(element)
+  }
+
+  editor.isVoid = (element) => {
+    return element.type === 'blank' ? true : isInline(element)
```

Comment:
```suggestion
    return element.type === 'blank' ? true : isVoid(element)
```
This should solve the link problem I think